StorageUtility/DeviceHandleInterface.h

```diff
@@ -0,0 +1,47 @@
+/*
+<License>
+Copyright 2015 Virtium Technology
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http ://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+</License>
+*/
+#ifndef __DeviceHandleInterface_h__
+#define __DeviceHandleInterface_h__
+#pragma once
+
+#include <map>
+#include <memory>
+
+#include "BasicTypes.h"
+#include "vtStorPlatformDefines.h"
+
+namespace vtStor
+{
+
+class cDeviceHandleInterface
+{
+public:
+    cDeviceHandleInterface();
+    virtual ~cDeviceHandleInterface();
+
+public:
+    DeviceHandle Handle();
+
+protected:
+    DeviceHandle m_DeviceHandle;
```

Comment:
You are right, interface must not define member or implementation
But I am confusing about some current implemented classes named as an interface but it is actually being like an abstract class
By the way, all new inplementation interfaces had been fixed